third_party/terraform/resources/resource_bigquery_table.go.erb

```diff
@@ -311,6 +311,19 @@ func resourceBigQueryTable() *schema.Resource {
 				MaxItems: 4,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"encryption_configuration": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"kms_key_name": {
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
There should be a `ForceNew: 1` here or in the encryption_configuration to enforce replacement of resource when the KMS key is exchanged. https://github.com/GoogleCloudPlatform/magic-modules/pull/2763#issuecomment-566918535

Reply:
It's meant to be above on the encryption_configuration, the key itself can be updated in-place.
See https://github.com/GoogleCloudPlatform/magic-modules/pull/2763#issuecomment-567148378.